codecov_cli/services/commit/__init__.py

```diff
@@ -43,11 +43,10 @@ def create_commit_logic(
 def send_commit_data(
     commit_sha, parent_sha, pr, branch, slug, token, service, enterprise_url
 ):
-    decoded_slug = decode_slug(slug)
-    pull_dict = get_pull(service, decoded_slug, pr) if not token else None
-    if is_fork_pr(pull_dict):
-        headers = {"X-Tokenless": pull_dict["head"]["slug"], "X-Tokenless-PR": pr}
-        branch = pull_dict["head"]["slug"] + ":" + branch
+    tokenless = os.environ.get("TOKENLESS")
```

Comment:
What's this?

Reply:
gonna leave a comment explaining this but this is how the CLI receives the username of the user to whom the fork belongs to and the branch name from the action

Reply:
that makes sense, a comment here would be pretty helpful thanks